docs/content/concepts/automation/schedules/examples.mdx

```diff
@@ -0,0 +1,751 @@
+---
+title: "Schedule examples | Dagster Docs"
+description: "Examples focused on Dagster schedules."
+---
+
+# Schedule examples
+
+This reference contains a variety of examples using Dagster [schedules](/concepts/paritions-schedules-sensors/schedules). Each example contains:
+
+- A summary
+- Additional notes
+- Links to relevant documentation
+- A list of the APIs used in the example
+
+---
+
+## Defining basic schedules
+
+The following examples demonstrate how to define some basic schedules.
+
+<TabGroup>
+  <TabItem name="Using ScheduleDefinition">
+
+This example demonstrates how to define a schedule using <PyObject object="ScheduleDefinition" /> that will run a job every day at midnight. While this example uses [op jobs](/concepts/ops-jobs-graphs/jobs) (<PyObject object="job" decorator />), the same approach will work with [asset jobs](/concepts/assets/asset-jobs) (<PyObject object="define_asset_job" />).
+
+```python file=concepts/partitions_schedules_sensors/schedules/schedules.py startafter=start_basic_schedule endbefore=end_basic_schedule
+@job
+def my_job(): ...
+
+
+basic_schedule = ScheduleDefinition(job=my_job, cron_schedule="0 0 * * *")
+```
+
+<table
+  className="table"
+  style={{
+    width: "100%",
+  }}
+>
+  <tbody>
+    <tr>
+      <td
+        style={{
+          width: "20%",
+        }}
+      >
+        <strong>Notes</strong>
```

Comment:
What happens if there are multiple long-form things that would be worthy of note?

Reply:
My intent is to keep these as short as possible since this content's goal isn't to teach. But if that were to be unavoidable, probably a bulleted list where each item starts with a bolded summary. Ex:

> - **Thing number one.** One fish, two fish, red fish, blue fish. Fish, fish, fish. 
> - **Thing number two.** I will not eat fish on a box, I will not eat them with a fox. I will not eat fish here or there, I will not eat them anywhere. Ew, ew ew.